Data/models/Device.swift

```diff
@@ -78,16 +78,16 @@ class Device: NSManagedObject, Syncable {
                 // Create
                 localDevice = add(context: context)
                 localDevice?.isCurrentDevice = true
-                DataController.saveContext(context: context)
+                DataController.save(context: context)
             }
             
             sharedCurrentDevice = localDevice
         }
         return sharedCurrentDevice
     }
     
-    class func deleteAll(completionOnMain: ()->()) {
-        let context = DataController.shared.workerContext
+    public class func deleteAll() {
```

Comment:
Is this function needed now that it is part of CRUD?

Reply:
The only difference between regular `deleteAll()` method is setting `fetchRequest.includesPropertyValues` to false, and `sharedCurrentDevice` to nil.